README.md

```diff
@@ -4,105 +4,132 @@ A logging plugin for [CakePHP](http://cakephp.org). The included `AuditableBehav
 
 The behavior tracks changes on two levels. It takes a snapshot of the fully hydrated object _after_ a change is complete and it also records each individual change in the case of an update action.
 
+Based on the Original AuditLog plugin for CakePHP 2.x found [here](https://github.com/robwilkerson/CakePHP-Audit-Log-Plugin) and off the awesome work of [rochamarcelo](https://github.com/rochamarcelo)
```

Comment:
seems irrelevant here, I don't mind attribution, but move it to the footer of the file you want it there - almost all of it has been rewritten multiple times since the original fork :) it's also visible in the "forked from" header for the repo - so a bit redundant